MacOSX/opensc-uninstall

```diff
@@ -20,6 +20,7 @@ rm -rf "/Applications/OpenSC Notify.app"
 rm -rf /Library/OpenSC
 rm -rf /Library/Security/tokend/OpenSC.tokend
 rm -rf /System/Library/Security/tokend/OpenSC.tokend
+rm -rf /System/Library/Frameworks/CryptoTokenKit.framework/PlugIns/OpenSCToken.appex
```

Comment:
Isn't the /System folder integrity protected by OS?
https://support.apple.com/en-us/HT204899

Reply:
I guess so. I did not find an other location where the extension could be put to be loaded at system startup. I guess the right way to work around SIP is to install OpenSCToken from the App Store...

Reply:
Why not create some dummy application? and install under /Library/OpenSC?

Reply:
The app would have to be loaded on system startup to register the plugin, right? So I'd also need some LaunchAgent. I hate it when solutions create new problems 😔

Reply:
**launch daemon**